### ISSUE TITLE
Can't connect to stremio

### ISSUE THREAD

**Issue by Jonybtw** [tymmesyde/peario-client#41]
**Describe the bug:**
When creating a room the error on the screenshot appears on every browser and I'm sure I'm not using an adblocker.

![1](https://user-images.githubusercontent.com/84144569/206525177-a832a71e-24d3-4689-b6be-6485d60dbbe2.png)

**To Reproduce:**
Create a room.

**Platforms:**
Desktop
Chrome, Firefox, Edge

**Comment by tymmesyde** [comment]
Are you sure that stremio is running and that you do not have any program that could be blocking the connection ? (Firewall, antivirus, ...)

**Comment by Jonybtw** [comment]
Yes, I've tried clearing cookies, disabling possible connection blocking extensions, different browsers and even on different devices.

**Comment by Cklinewv2** [comment]
Also having this problem

**Comment by MrKospe** [comment]
I'm using a MacBook Air M3 from 2024. I have installed Stremio and logged in to my account. I have turned off the built-in Firewall from Mac; I have disabled all content blockers; I have no Antivirus or AntiMalware app installed and have no VPN connection turned on. I have also installed the addons in the Peario website. Screenshots at the end of the text.

Nonetheless, Peario can't connect to Stremio. I have found no answer online.

Error message that keeps popping up:
<img width="527" height="117" alt="Image" src="https://github.com/user-attachments/assets/a4369a7d-b5b8-46f0-9959-328740d9f137" />

All proof of what I said above

Logged in in the browser website:
<img width="1303" height="679" alt="Image" src="https://github.com/user-attachments/assets/a73d8de2-6751-448a-831a-254158904eb3" />

Logged in in the app:
<img width="1191" height="728" alt="Image" src="https://github.com/user-attachments/assets/f5b944a9-1597-48c0-affe-b135c76fb6de" />

Nothing blocking the peario website
<img width="1344" height="831" alt="Image" src="https://github.com/user-attachments/assets/ca7fc658-9a74-4228-b7c5-fbba24d532a6" />

Addons installed:
<img width="1351" height="848" alt="Image" src="https://github.com/user-attachments/assets/7fa75165-18ad-41bf-a59b-dd265560c42c" />

Network settings of my Mac:
<img width="733" height="635" alt="Image" src="https://github.com/user-attachments/assets/d6421159-f2e1-4696-9fe3-8c84e7f3aa38" />
